uaclient/entitlements/fips.py

```diff
@@ -195,8 +247,14 @@ def install_packages(
 
         for pkg in desired_packages:
             try:
-                super().install_packages(
-                    package_list=[pkg], cleanup_on_failure=False, verbose=False
+                apt.run_apt_install_command(
```

Comment:
Just double checking, why are we changing this ?

Reply:
Every time super().install_packages is called, the kernel downgrade warning is sent.  In my testing, we generally saw it three times.  Grant suggested that this call should properly be `run_apt_install_command` instead.